backend/index.js

```diff
@@ -10,6 +10,22 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/yelp',(req,res)=>{
+
+    let city = "dallas"
```

Comment:
While hardcoding the city is good for practice, the city will be the zip code that the user wants to go to. The zip code should be coming in as part of the post request. So the incoming body will be something like { 'zip':'11111'}. That means that youll probably have to change this to a post request so that it can have a body. Another option is to include the zip code in the url. Like [this](https://github.com/Codubee/Node-Js-API/blob/axios-api-call/hello.js#L37) example